frontend_tests/casper_tests/06-settings.js

```diff
@@ -262,8 +262,8 @@ casper.waitUntilVisible('#default_language_modal');
 
 casper.thenClick('a[data-code="zh-hans"]');
 
-casper.waitUntilVisible('#display-settings-status', function () {
-    casper.test.assertSelectorHasText('#display-settings-status', '简体中文 is now the default language');
+casper.waitUntilVisible('#language-settings-status', function () {
```

Comment:
changing `'#language-settings-status'` to `'#language-settings-status a'` can make this works and same for its other instances.